website/docusaurus.config.js

```diff
@@ -187,7 +187,7 @@ module.exports = {
           ],
         },
       ],
-      copyright: `Copyright © ${new Date().getFullYear()} AutoGen Authors |  <a target="_blank" style="color:#10adff" href="https://go.microsoft.com/fwlink/?LinkId=521839">Privacy and Cookies</a>`,
+      copyright: `Copyright © ${new Date().getFullYear()} AutoGen Authors |  <a target="_blank" style="color:#10adff" href="https://go.autogen-ai.com/fwlink/?LinkId=521839">Privacy and Cookies</a>`,
```

Comment:
can we have the uri pointing to our governance repository, we can add the privacy policies.

cc @sonichi @qingyun-wu